alarm_req_listener.cpp

```diff
@@ -123,11 +126,25 @@ bool AlarmReqListener::zmq_init_sck()
     return false;
   }
 
-  std::stringstream ss;
-  ss << "tcp://127.0.0.1:" << ZMQ_PORT; 
+  std::string sf = std::string("/var/run/clearwater/alarms");
+  std::string ss = std::string("ipc://" + sf);
+  snmp_log(LOG_INFO, "AlarmReqListener: ss='%s'", ss.c_str());
 
   int rc;
-  while (((rc = zmq_bind(_sck, ss.str().c_str())) == -1) && (errno == EINTR))
+  rc=remove(sf.c_str());
+  if (rc == -1)
+  {
+    if (errno != ENOENT)
+    {
+      snmp_log(LOG_ERR, "remove(%s) failed: %s - killing myself", sf.c_str(), strerror(errno));
+      kill(getpid(), SIGKILL);
```

Comment:
Bypassing the snmpd exit handlers is a bold move. Can you provide some more detail on why this is required?